redisinsight/ui/src/pages/rdi/pipeline-management/components/header/RdiPipelineHeader.tsx

```diff
@@ -53,106 +45,68 @@ const RdiPipelineHeader = () => {
       event: TelemetryEvent.RDI_DEPLOY_CLICKED,
       eventData: {
         id: rdiInstanceId,
-        jobsNumber: values?.jobs?.length,
-      },
+        jobsNumber: values?.jobs?.length
+      }
     })
     setIsPopoverOpen(true)
   }
 
   return (
-    <div className={cx(styles.container)}>
-      <EuiFlexGroup style={{ height: '100%' }} gutterSize="none" alignItems="center" responsive={false}>
-        <EuiFlexItem style={{ overflow: 'hidden' }}>
-          <div className={styles.breadcrumbsContainer} data-testid="breadcrumbs-container">
-            <div>
-              <EuiToolTip
-                position="bottom"
-                content="My RDI instances"
+    <Header
+      actions={(
+        <EuiOutsideClickDetector onOutsideClick={handleClosePopover}>
```

Comment:
oh, i see what you did. We need to think about rdi templates (i global template and one for pipeline management)